EIPS/eip-7092.md

```diff
@@ -0,0 +1,342 @@
+---
+eip: 7092
+title: Financial Bonds
+description: This interface defines a specification for financial bonds tokenization
+author: Samuel Gwlanold Edoumou (@Edoumou)
+discussions-to: https://ethereum-magicians.org/t/financial-bonds/14461
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-05-28
+---
+
+## Abstract
+
+The proposed standard allows for the implementation of basic functionality for fixed income financial bonds with smart contracts.
+Principal bonds characteristics such as the bond isin, the issue volume, the issue date, the maturity date, the coupon rate,
+the coupon frequency, the principal, or the day count basis are defined to allow issuing bonds in the primary market (origination),
+and different transfer functions allow to buy or sell bonds in the secondary market. The standard also providses a functionality to
+allow bonds to be approved by owners in order to be spent by third party.
+
+## Motivation
+
+Fixed income instruments are one of the asset classes that is widely used by corporations and other entities to raise funds. Bonds
+are considered more secured than equity since the issuer is supposed to repay the principal at maturity in addition to coupons
+that are paid to investsors.
+
+This standard interface allows fixed income instruments to be represented as on-chain tokens, so as they can be managed through wallets,
+and be used by applications like decentrailized exchanges.
+
```

Comment:
Are there any existing standards for bonds on-chain? If so, you should note why this standard is better.